MultiPartyExample/MultiPartyViewController.swift

```diff
@@ -89,9 +91,15 @@ class MultiPartyViewController: UIViewController {
         let audioIndicatorOrigin = CGPoint(x: layoutFrame.minX + videoViewWidth - audioIndicatorSize.width - MultiPartyViewController.kAudioIndicatorPadding,
                                            y: videoViewHeight + topY - audioIndicatorSize.height - MultiPartyViewController.kAudioIndicatorPadding)
 
-        // Layout remote Participants
         localParticipantAudioIndicator.frame = CGRect(origin: audioIndicatorOrigin, size: audioIndicatorSize)
 
+        let networkQualityLevelIndicatorSize = networkQualityLevelIndicator.intrinsicContentSize
+        let networkQualityLevelIndicatorOrigin = CGPoint(x: layoutFrame.minX + videoViewWidth - networkQualityLevelIndicatorSize.width - MultiPartyViewController.kNetworkQualityIndicatorPadding,
+                                                         y: topY + MultiPartyViewController.kNetworkQualityIndicatorPadding)
+
+        networkQualityLevelIndicator.frame = CGRect(origin: networkQualityLevelIndicatorOrigin, size: networkQualityLevelIndicatorSize)
```

Comment:
It would be nice if we had used `localParticipantVideoView` 's frame to calculate where to put the indicator.

Also I am assuming you have tried the case when there is no local video track.

Reply:
Another option is we can use storyboard auto layout. up to you.

Reply:
The layout for LocalParticipant is pretty rough right now, and it needs to be in its own UIView. At the moment the indicator for mute has no guarantee of being aligned with the video itself.